devel/gaa/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem               1.0
+PortGroup                github 1.0
+
+github.setup             cooljeanius gaa 1.6.6_1 v
+revision                 3
+platforms                darwin
+categories               devel
+maintainers              gwmail.gwu.edu:egall openmaintainer
+description              ${name} Argument Analyser
+license                  GPL-2+
+long_description         ${name} simplifies the life of the programmer: \
+                         you do NOT have to worry about the arguments \
+                         given to it! A short text written in GAA \
+                         language generates C-code which analyses \
+                         the arguments and creates your program's help.
+
+homepage                 http://${name}.sf.net
+
+fetch.type               git
+git.url                  https://github.com/cooljeanius/${name}.git
```

Comment:
It is redundant to specify `git.url` as that has already been set by `github.setup`. This line can be removed.

Reply:
Removed in [3e0470f](https://github.com/macports/macports-ports/pull/20972/commits/3e0470fc5ad30184fec83c23722be7b056bbd219).